README.rst

```diff
@@ -7,13 +7,13 @@ Welcome to Pyccel
 
 The aim of **Pyccel** is to provide a simple way to generate automatically, parallel low level code. The main uses would be:
 
-1. Convert a *Python* code (or project) into a Fortran
+1. Convert a *Python* code (or project) into a Fortran or C
```

Comment:
Should be `into a Fortran or C code` or  `into Fortran or C`

Reply:
done